README.md

```diff
@@ -3,7 +3,9 @@
 **You are currently viewing a development branch**  
 Uses PUPPI Jets as default for Run3.
 
-Tested with 2022 data (ReRecoCDE and PromptRecoFG), MC for Run3 (Run3Summer22 and Run3Summer22EE, nanoAODv11).
+This branch runs with 2022 data (ReRecoCDE and PromptRecoFG), MC for Run3 (Run3Summer22 and Run3Summer22EE, nanoAODv11),
+in 13_0_X.
+This branch reruns Puppi v17, reclusters the AK8 Puppi and AK8 taggers, and then reruns the new AK4 taggers (new DeepJet, new ParticleNetAK4 and RobustParTAK4) available from 13_0_X.
```

Comment:
Could you provide me the created root files from running the cmsRun commands locally? I can then create updated htmls for size and content of PFNano